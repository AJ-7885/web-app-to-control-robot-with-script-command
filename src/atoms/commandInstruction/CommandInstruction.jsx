import React from 'react';
import { ExplainPanel } from './CommandInstruction.style';

export const CommandInstruction = () => (
  <ExplainPanel>
    Commands: <br />
    <br />
    1. FORWARD &quot;NUMBER VALUE&quot;
    <br />
    2. WAIT
    <br />
    3. POSITION &quot;X value&quot; &quot;Y value&quot; &quot;direction
    (WEST,EAST,NORTH,SOUTH)&quot;
    <br />
    4. LEFT , RIGHT , UP . DOWN
    <br />
    5. TURNAROUND
  </ExplainPanel>
);
