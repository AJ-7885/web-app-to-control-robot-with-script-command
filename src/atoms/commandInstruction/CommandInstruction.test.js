import React from 'react';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import { CommandInstruction } from './CommandInstruction';

test('renders component and check the content', () => {
  const { getByText } = render(<CommandInstruction />);
  const element = getByText(/4. LEFT , RIGHT , UP . DOWN/i);
  expect(element).toBeInTheDocument();
});

test('renders correctly', () => {
  const component = renderer.create(<CommandInstruction />).toJSON();
  expect(component).toMatchSnapshot();
});
