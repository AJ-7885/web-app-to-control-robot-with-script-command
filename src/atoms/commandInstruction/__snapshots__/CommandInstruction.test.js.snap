// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-AxjAm jgCQpp"
>
  Commands: 
  <br />
  <br />
  1. FORWARD "NUMBER VALUE"
  <br />
  2. WAIT
  <br />
  3. POSITION "X value" "Y value" "direction (WEST,EAST,NORTH,SOUTH)"
  <br />
  4. LEFT , RIGHT , UP . DOWN
  <br />
  5. TURNAROUND
</div>
`;
